src/app/components/TutorialDialog/screen2/index.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useState, useEffect } from 'react';
+import { translations } from 'locales/i18n';
+import { useTranslation } from 'react-i18next';
+import arm1 from 'assets/images/tutorial/arm_1.svg';
+import arm2 from 'assets/images/tutorial/arm_2.svg';
+import stepBox from 'assets/images/tutorial/step-box.svg';
+import leftArrow from 'assets/images/tutorial/left_arrow.svg';
+import rightArrow from 'assets/images/tutorial/right_arrow.svg';
+import engage from 'assets/images/tutorial/engage.svg';
+import speechBubble from 'assets/images/tutorial/speech_bubble.svg';
+import leftBox from 'assets/images/tutorial/left_box.svg';
+import rightBox from 'assets/images/tutorial/right_box.svg';
+import badgerBody from 'assets/images/tutorial/badger_body.svg';
+import crater from 'assets/images/tutorial/crater.svg';
+import { Icon } from '@blueprintjs/core';
+import { useContent } from '../../../hooks/tutorial/useContent';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+interface Props {
+  onNetwork: boolean;
+  mouseLeave: boolean;
+  activeTutorial: boolean;
+  handleEngage: () => void;
+}
+
+export function Screen2(props: Props) {
+  const { t } = useTranslation();
+  const content = useContent();
+  const [cycle, setCycle] = useState(false);
+  const [step, setStep] = useState(props.activeTutorial === true ? 7 : 1);
+  const [speechText, setSpeechText] = useState(
+    props.activeTutorial ? content[7].speech : content[1].speech,
+  );
+  const [moving, setMoving] = useState(false);
```

Comment:
`setMoving` is unused, consider commenting it out or remove entire state if it doesn't do anything.